requirements.txt

```diff
@@ -1,9 +1,10 @@
 boto3>=1.4.3
 botocore>=1.4.93
 hvac>=0.8.1
-PyYaml
+PyYaml>=5.3
 mysql-connector-python
 pyminifier
 funcparserlib
 iso8601
 colorama
+importlib_resources
```

Comment:
Derek's heaviside `requirements.txt` specifies this like so:
`importlib_resources; python_version < "3.7"`

Reply:
Actually, maybe we want to make soft link to the `requirements.txt` in the heaviside submodule folder so it's clear that certain dependencies are only included due to that submodule.

`heaviside-requirements.txt -> lib/heaviside.git/requirements.txt`

Reply:
I like that approach too.

Reply:
You got it!